docs/examples/ratify-verify-azure-cmd.md

```diff
@@ -1,296 +1,361 @@
-tar xvzf notation.tar.gz -C  /usr/local/bin notation
-```
-
-### ORAS
-
-Install ORAS 0.16.0 on a Linux machine. You can refer to the [ORAS installation guide](https://oras.land/cli/) for details.
-
-```bash
-# Download the ORAS binary
-curl -Lo oras.tar.gz https://github.com/oras-project/oras/releases/download/v0.16.0/oras_0.16.0_linux_amd64.tar.gz
-# Extract it from the binary and copy it to the bin directory
-tar xvzf oras.tar.gz -C /usr/local/bin oras
-```
+In this article:
 
-### Ratify
 
-Install Ratify v1.0.0-beta.2 from [Ratify GitHub Release](https://github.com/deislabs/ratify/releases/tag/v1.0.0-beta.2).
+* Create and store a signing certificate in Azure Key Vault
+* Sign a container image with notation
+* Create an AKS cluster and ACR registry with Azure Workload Identity configured
+* Intall Ratify and Gatekeeper
+* Validate a container image signature with Ratify and Gatekeeper
+* Deploy a signed image to AKS
 
-```bash
-# Download the Ratify binary
-RATIFY_VERSION=1.0.0-beta.2
-curl -Lo ratify.tar.gz https://github.com/deislabs/ratify/releases/download/v${RATIFY_VERSION}/ratify_${RATIFY_VERSION}_Linux_amd64.tar.gz
-# Extract it from the binary and copy it to the bin directory
-tar xvzf ratify.tar.gz -C /usr/local/bin ratify
-```
-
-## Presets
-
-### Set up ACR and Auth information
-
-```bash
-export ACR_NAME=<YOUR_ACR_NAME>
-export REGISTRY=$ACR_NAME.azurecr.io
-export REPO=${REGISTRY}/net-monitor
-export IMAGE=${REPO}:v1
-export RESOURCE_GROUP=$ACR_NAME-acr
-export LOCATION=westus3
-
-# Create an ACR
-# Premium to use tokens
-az group create -n $RESOURCE_GROUP -l $LOCATION
-az acr create -n $ACR_NAME -g $RESOURCE_GROUP --sku Premium
-az configure --default acr=$ACR_NAME
-az acr update --anonymous-pull-enabled true
-
-# Using ACR Auth with Tokens
-export NOTATION_USERNAME='wabbitnetworks-token'
-export NOTATION_PASSWORD=$(az acr token create -n $NOTATION_USERNAME \
-                    -r $ACR_NAME \
-                    --scope-map _repositories_admin \
-                    --only-show-errors \
-                    -o json | jq -r ".credentials.passwords[0].value")
-
-docker login $REGISTRY -u $NOTATION_USERNAME -p $NOTATION_PASSWORD
-oras login $REGISTRY -u $NOTATION_USERNAME -p $NOTATION_PASSWORD
-```
+![](https://i.imgur.com/1f9dfV9.png)
 
-## Demo 1: Discover & Verify Signatures using Ratify
 
-### Sign the image using ```notation```
+## Install the Notation CLI and AKV plugin
 
-1. We will build, push, sign the image in ACR
+1. Install Notation v1.0.0-rc.2 on a Linux environment. You can also download the package for other environments from the [release page](https://github.com/notaryproject/notation/releases/tag/v1.0.0-rc.2).
 
     ```bash
-    # Build, push, sign the image in ACR
-    echo $IMAGE
+    # Download, extract and install
+    curl -Lo notation.tar.gz https://github.com/notaryproject/notation/releases/download/v1.0.0-rc.2/notation_1.0.0-rc.2_linux_amd64.tar.gz
```

Comment:
```suggestion
    export NOTATION_VERSION=1.0.0-rc.2
    curl -Lo notation.tar.gz https://github.com/notaryproject/notation/releases/download/v${NOTATION_VERSION}/notation_${NOTATION_VERSION}_linux_amd64.tar.gz
```

so it's easy to update these in the future (applies to others as well like notation-akv, etc)

Reply:
Accepted and changed.